src/interrupt.c

```diff
@@ -472,11 +473,84 @@ void unregister_SP_handler( void (*callback)() )
     __unregister_callback(&SP_callback,callback);
 }
 
+
+/**
+ * @brief Register a timer callback
+ * 
+ * The callback will be used when the timer interrupt is triggered by the CPU.
+ * This happens when the COP0 COUNT register reaches the same value of the
+ * COP0 COMPARE register.
+ * 
+ * This function is useful only if you want to do your own low level programming
+ * of the internal CPU timer and handle the interrupt yourself. In this case,
+ * also remember to activate the timer interrupt using
```

Comment:
The end of the sentence is missing. Thanks for the additional documentation!

Reply:
Done